### PR TITLE
Add Lazy texture caching Tip

### PR DESCRIPTION
**Pull request by sum2012**

### CODE REVIEW

File: UI/GameSettingsScreen.cpp [hunk 1]
```diff
@@ -449,6 +449,10 @@ void GameSettingsScreen::CreateViews() {
 	});
 
 	CheckBox *texBackoff = graphicsSettings->Add(new CheckBox(&g_Config.bTextureBackoffCache, gr->T("Lazy texture caching", "Lazy texture caching (speedup)")));
+	texBackoff->OnClick.Add([=](EventParams& e) {
+		settingInfo_->Show(gr->T("Lazy texture caching Tip", "When enable,game font may have problem"), e.v);
```
Comment (jcchikikomori) [inline UI/GameSettingsScreen.cpp:453]: This might be a cosmetic, but change this to
`When enabled, game font may have problem`

**Comment by ResurrectionRedmi9** [comment]
Why this is still not merge? this is just a tip about lazy texture caching.

**Comment by hrydgard** [comment]
Because the typo feedback from jcchikikomori has still not been applied.

**Comment by hrydgard** [comment]
Went ahead and did this one in df9be77. Closing.

**Pull request closed without merging**